meta/runtime.yml

```
@@ -202,6 +202,14 @@ plugin_routing:
       tombstone:
         removal_version: 10.0.0
         warning_text: Use community.general.consul_token and/or community.general.consul_policy instead.
+    dimensiondata_network:
+      deprecation:
+        removal_version: 13.0.0
+        warning_text: Service and its endpoints are not longer available.
```

Isn't it *no* longer available? Though I guess both are fine.
```suggestion
        warning_text: Service and its endpoints are no longer available.
```

---

oopsie daisy
fixed